string a_operation_landing_reason
---
int64 a_operation_status
